installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -0,0 +1,81 @@
+[id="installing-openstack-user"]
+= Installing a cluster on OpenStack on your own infrastructure
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-user
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+{rh-openstack-first} that runs on infrastructure that you provide.
+
+Using your own infrastructure allows you to integrate your cluster with existing infrastructure and modifications. The process requires more labor on your part than installation-program-provisioned installations, however, as you must create all {rh-openstack} resources, like Nova servers, Neutron ports, and security groups.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* Have access to an {rh-openstack} administrator's account
```

Comment:
When I read "OpenStack administrator's account", I think of one that can add and remove services to the catalog, change quotas for OpenStack users, integrate new hardware into the OpenStack cluster. This level of privilege is not required: typically, you can install ShiftStack as a regular customer of a public cloud.

In other words: for UPI, you need (at minimum) the same exact access to OpenStack that you would need for IPI.